spec/models/pyramidal_tiff_spec.rb

```diff
@@ -88,7 +88,7 @@
       .to_return(status: 200, body: File.open('spec/fixtures/images/sample_cmyk.tiff', 'rb'))
     ptiff_tmpdir = "spec/fixtures/images/ptiff_images/"
     allow(described_class).to receive(:new).and_return(ptf)
-    allow(ptf).to receive(:convert_to_ptiff).and_return(ptf.convert_to_ptiff("spec/fixtures/images/sample_cmyk.tiff", ptiff_tmpdir))
+    allow(ptf).to receive(:convert_to_ptiff).and_return(ptf.convert_to_ptiff(__FILE__, ptiff_tmpdir))
```

Comment:
doesn't this mean you're trying to convert this file in to a ptiff now? how does that work?

Reply:
This is in the context of a test that asks "if I can't turn file x into a ptiff, does the application still behave appropriately" and the old "file x" can now be turned into a ptiff (good!) but now we need a new file that *can't* be turned into a ptiff in order for the test to pass.